backend/uclapi/resources/views.py

```diff
@@ -0,0 +1,45 @@
+import os
+import requests
+import xml.etree.ElementTree
+
+from rest_framework.decorators import api_view
+
+from roombookings.decorators import does_token_exist
+
+from roombookings.helpers import PrettyJsonResponse as JsonResponse
+
+
+@api_view(["GET"])
+@does_token_exist
+def get_pc_availability(request):
+    r = requests.get(os.environ.get("PCA_LINK"))
```

Comment:
This will probably trigger @wilhelmklopp :D
We should use `os.environ["PCA_LINK"]` because otherwise we'll try to do a request for None if the env variable is not configured. We should allow this exception to get raised so that it's picked up by Sentry.

Reply:
lalala this again comes to this question. do we need to worry about us setting the configs? I have edited it to not catch that error. But I am not sure if this is what you want.